terraform/eks_adot_operator_cluster_setup/main.tf

```diff
@@ -20,6 +20,12 @@ terraform {
       version = ">= 1.7.0"
     }
   }
+  ## comment out the backend block if you want the terraform state file to be locally saved on the computer (or) change the name of the s3 bucket if you are testing out on your personal AWS account
```

Comment:
Is this the only way to save the state file to a local machine after this change is merged? I am very much opposed to leaving in instructions to `comment out` code. This will greatly increase changes that we will have commented code merged back into the codebase. 

Shouldn't every user be saving the state file remotely even if it to a personal account? There are cost associations but so are the other infra assets that are being deployed.

Reply:
Are any other solutions possible? Toggle the saving of the file through a user provided variable? Maybe just always saving the file remotely but instructing users on how to pull the file from s3?

Reply:
I think its recommended to always have the state file saved on the s3 bucket. I don't see a way to disable the back-end block through a variable. We can give instructions to the user on curl the latest version of the state file from s3 bucket or reverting back to an older version.

Reply:
Sure, I would agree with that. We should remove this comment then.